libraries/AP_HAL_Linux/RCOutput_Multi.cpp

```diff
@@ -0,0 +1,186 @@
+#include "RCOutput_Multi.h"
+#include <stdio.h>
+#include <AP_HAL/AP_HAL.h>
+
+#define MIN(a,b) ((a)<(b)?(a):(b))
```

Comment:
Use the one from AP_Math.h; it doesn't have the re-eval bug this formulation has

Reply:
done